.github/workflows/Rubocop.yml

```diff
@@ -0,0 +1,15 @@
+name: Rubocop
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Would it make sense to use Chromebrew on Chrome OS Docker images?

Reply:
Should not be necessary. Standard ubuntu images should be more robust for this sort of thing...

Reply:
We'd have to self-host the images.

Reply:
That is fine.  We can add to a separate repo here dedicated to that if we want.  @satmandu, thoughts?

Reply:
If we do do that, then we can add actions for testing package modifications, so I do like that idea.